content/changelog/2024-08-23.md

```diff
@@ -0,0 +1,57 @@
+
+As with all of our Beta features, improvements are ongoing. If you have any feedback, we'd love to hear it. Let us know via the [Feedback](https://console.neon.tech/app/projects?modal=feedback) form in the Neon Console or our [feedback channel](https://discord.com/channels/1176467419317940276/1176788564890112042) on Discord.
+
+### Early Access to Neon Organizations
+
+We're also very happy to announce that Neon Organizations are now available for members of our Early Access Program.
+
+![get started with your new org](/docs/relnotes/orgs_create_next.png)
+
+Create a new organization, transfer over your projects, invite your team and get started collaborating. [Join now](https://neon.tech/early-access-program) to get a first look at Neon Organizations and other upcoming features before they go live.
+
+See [Neon Organizations](/docs/manage/organizations) to learn more.
+
+### Console enhancements
+
+We've made several enhancements to the Neon Console to improve your experience and streamline your interaction with our user interface.
+
+- You can now access Neon projects shared with you from the project breadcrumb selector in the Neon Console.
+  ![Project breadcrumb selector](/docs/relnotes/breadcrumb_project_selector.png 'no-border')
+- We added a **Created by** column to the **Branches** page in the Neon Console to display the branch creation source. For example, this new column shows if a branch was created in the console or by the Neon Vercel Integration. Additionally, for branches created by the Neon Vercel Integration, a pop-up is displayed when you hover, with your GitHub account name, a Vercel Preview link, and a link to the pull request associated with the preview branch.
+  ![Branch created by column](/docs/relnotes/branch_created_by_column.png 'no-border')
+- The **Billing** page for Free Plan users now shows a percentage value for branch compute usage.
+  ![Compute branches metric](/docs/relnotes/compute_branches.png 'no-border')
+- We've added a new example to the Neon Console Quickstart showing how to connect to to your Neon database from NestJS. You can access the Quickstart from your Project Dashboard by selecting **Show quickstart**.
+  ![NestJS example](/docs/relnotes/nestjs.png)
+- The **Primary compute** column on the Branches page now shows your configured autoscaling range, replacing the `Autoscales` badge that was shown previously.
+  ![Autoscaling range](/docs/relnotes/autoscaling_range.png 'no-border')
+- We now support self-serve account deletion should you need to remove your Neon account for any reason. See [Delete your account](/docs/manage/accounts#delete-your-account) for details.
```

Comment:
This link doesn't work for me

Reply:
Let's remove the delete item for now. Not sure it's going to get released today.

Reply:
Removed for today's changelog. Doesn't look like it will make it in.